contrib/abcl-introspect/procyon.lisp

```diff
@@ -4,9 +4,22 @@
      #:disassemble-class-bytes))
 (in-package :abcl-introspect/jvm/tools/procyon)
 
+
+(defun bytes->temp-file (bytes)
+  (let* ((create-temp-file-method
+           (java:jmethod "java.io.File" "createTempFile"
```

Comment:
@alejandrozf Note that there already exists a `ext:make-temp-file` interface to File.createTempFile() which could have been re-used.  Not a huge problem at all:  just pointing this out for next time.

Reply:
Oh. I see. Thank you @easye